### ISSUE TITLE
Create optional --reuse-password flag for decrypting keys in validator client

### ISSUE THREAD

**Issue by ethDreamer** [sigp/lighthouse#1881]
## Description

The validator client currently has a --reuse-password flag that allows it to reuse the same password when importing keys. However, it does NOT have this feature during normal operation when decrypting keys that don't have a password stored in validator_definitions.yml. Without this feature, running multiple validators without the password stored is way too tedious.

**Comment by dankrad** [comment]
I have come around to the conclusion that it's a good option not to have passwords stored on disk. You lose the ability to reboot but make it much harder for someone to get validator keys by getting physical access.

**Comment by paulhauner** [comment]
This might be a good spare-time task for @pawanjay176 :slightly_smiling_face:

**Comment by ethDreamer** [comment]
Sorry @pawanjay176 I only just saw now that you had assigned this to yourself >.<
Not sure how far you went on this? Hopefully we didn't waste too much duplicate work..

**Comment by pawanjay176** [comment]
Ohh sweet. I had forgotten about this guy. Thanks for finishing this up :)

**Comment by ellamonroe6047** [comment]
Yes, there’s some kind of givеаway going on. I got some Ethereum there yesterday for different wallets. It’s unlikely they’ll take it back from me, but it’s a pity I didn’t have more wallets and couldn’t get more. But maybe it’s for the best.

![Image](https://github.com/user-attachments/assets/06768049-7409-4b1a-aa6e-60b59d8f46bd)
